syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -181,6 +181,13 @@ func (j *archiveParser) discoverMainPackage(ctx context.Context) (*pkg.Package,
 		return nil, nil
 	}
 
+	// check for existence of Weave-Classes manifest key in order to exclude jars getting misrepresented as
+	// their targeted counterparts, e.g. newrelic spring and tomcat instrumentation
+	if _, ok := manifest.Main.Get("Weave-Classes"); ok {
+		log.Debugf("excluding archive due to Weave-Classes manifest entry: %s", j.location)
+		return nil, nil
```

Comment:
Is it necessary to skip matching on this jar entirely, or is there a similar heuristic where we could correctly find the NewRelic jar?

Reply:
We _are_ finding the newrelic jar, these are subcomponents of it. E.g.:
```
$go run ./cmd/syft ~/Downloads/newrelic-java-8.9.1.zip 
NAME                       VERSION  TYPE                         
agent-bridge               8.9.1    java-archive                  
agent-bridge-datastore     8.9.1    java-archive                  
newrelic                   8.9.1    java-archive                  
newrelic-api               8.9.1    java-archive  (+1 duplicate)  
newrelic-api-javadoc                java-archive                  
newrelic-api-sources                java-archive                  
newrelic-security-agent    1.1.0    java-archive                  
newrelic-security-api      1.1.0    java-archive                  
newrelic-weaver-api        8.9.1    java-archive                  
newrelic-weaver-scala-api  8.9.1    java-archive  
```
I don't think it's valuable to try to identify these separately.

Reply:
Oh I see.

I misunderstood the comment at https://github.com/anchore/syft/pull/2796/files#diff-0965089b69371402427667e60ad9d7ad5698ae0b330b5c7430e2b0c671655fe5R1356 `// we expect no packages to be discovered from this` - I thought that meant we wouldn't find the New Relic jar at all.

What is meant by that comment?

Reply:
The test is for a manifest with the `Weave-Classes` attribute being excluded. I updated the comment to hopefully be more clear.

Reply:
I should note I can't seem to find any vulnerabilities reported against the Java agent or related components anywhere, though. But it seems that the vulnerabilities reported are not against individual components but rather the platform: https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=newrelic

Reply:
Also note, prior to this change, this is what we see from that distribution -- almost all of these packages are the instrumentation jars:
```
NAME                                   VERSION            TYPE                         
agent-bridge                           8.9.1              java-archive                  
agent-bridge-datastore                 8.9.1              java-archive                  
akka                                   2.2-1.0            java-archive                  
akka-http                              2.11_2.4.5-1.0     java-archive  (+1 duplicate)  
akka-http                              2.13_10.1.8-1.0    java-archive                  
akka-http-core                         0.4-1.0            java-archive                  
akka-http-core                         0.7-1.0            java-archive                  
akka-http-core                         1.0-1.0            java-archive                  
akka-http-core                         10.0-1.0           java-archive  (+1 duplicate)  
akka-http-core                         10.2.0-1.0         java-archive                  
akka-http-core                         2.11_10.0.11-1.0   java-archive  (+1 duplicate)  
akka-http-core                         2.13_10.1.8-1.0    java-archive                  
akka-http-core                         2.13_10.2.0-1.0    java-archive  (+1 duplicate)  
anorm                                  2.3-1.0            java-archive                  
anorm                                  2.4-1.0            java-archive                  
apache-ldap                            1.0                java-archive                  
apache-log4j                           1-1.0              java-archive                  
apache-log4j                           2.0-1.0            java-archive                  
apache-log4j                           2.11-1.0           java-archive                  
apache-log4j                           2.17.2-1.0         java-archive                  
apache-log4j                           2.6-1.0            java-archive                  
apache-log4j                           3.0.0-1.0          java-archive                  
apache-struts                          2.0-1.0            java-archive                  
async-http-client                      2.0.0-1.0          java-archive  (+1 duplicate)  
async-http-client                      2.1.0-1.0          java-archive                  
aws-java-sdk-dynamodb                  1.11.106-1.0       java-archive                  
aws-java-sdk-dynamodb                  2.15.34-1.0        java-archive                  
aws-java-sdk-s3                        1.2.13-1.0         java-archive                  
aws-java-sdk-s3                        2.0-1.0            java-archive                  
aws-java-sdk-sns                       1.11.12-1.0        java-archive                  
aws-java-sdk-sns                       2.0-1.0            java-archive                  
aws-java-sdk-sqs                       1.10.44-1.0        java-archive                  
aws-java-sdk-sqs                       2.1.0-1.0          java-archive                  
aws-wrap                               0.7.0-1.0          java-archive                  
camel-xpath                            1.0                java-archive                  
cassandra-datastax                     3-1.0              java-archive                  
cassandra-datastax                     3.0.0-1.0          java-archive                  
cassandra-datastax                     3.8.0-1.0          java-archive                  
cassandra-datastax                     4-1.0              java-archive                  
cassandra-datastax                     4.0.0-1.0          java-archive                  
cats-effect                            2-1.0              java-archive                  
cats-effect                            3-1.0              java-archive                  
cats-effect                            3.3.0-1.0          java-archive                  
commons-jxpath                         1.0                java-archive                  
completable-future-jdk8                1.0                java-archive                  
completable-future-jdk8u40             1.0                java-archive                  
cxf                                    2.7-1.0            java-archive                  
dynamodb                               1.11.390-1.0       java-archive                  
dynamodb                               1.11.453-1.0       java-archive                  
dynamodb                               1.11.459-1.0       java-archive                  
dynamodb                               1.11.80-1.0        java-archive                  
dynamodb                               2.1.0-1.0          java-archive                  
dynamodb                               2.1.2-1.0          java-archive                  
dynamodb                               2.15.34-1.0        java-archive                  
ejb                                    3.0-1.0            java-archive                  
ejb                                    4.0-1.0            java-archive                  
file-low-priority-instrumentation      1.0                java-archive                  
file-operation                         1.0                java-archive                  
glassfish                              3-1.0              java-archive                  
glassfish                              6-1.0              java-archive                  
glassfish-jmx                          1.0                java-archive                  
graalvm-jsinjection                    19.0.0-1.0         java-archive                  
graalvm-jsinjection                    22.0.0-1.0         java-archive                  
grails                                 1.3-1.0            java-archive                  
grails                                 2-1.0              java-archive                  
grails-async                           2.3-1.0            java-archive                  
graphql-java                           16.2-1.0           java-archive                  
graphql-java                           17.0-1.0           java-archive                  
graphql-java                           21.0-1.0           java-archive                  
grpc                                   1.22.0-1.0         java-archive  (+1 duplicate)  
grpc                                   1.30.0-1.0         java-archive                  
grpc                                   1.4.0-1.0          java-archive  (+1 duplicate)  
grpc                                   1.40.0-1.0         java-archive  (+1 duplicate)  
hibernate                              3.3-1.0            java-archive                  
hibernate                              3.5-1.0            java-archive                  
hibernate                              4.0-1.0            java-archive                  
hibernate                              4.2-1.0            java-archive                  
hibernate                              4.3-1.0            java-archive                  
http-async-client                      4-1.0              java-archive  (+1 duplicate)  
http4s-blaze-client                    2.12_0.21-1.0      java-archive                  
http4s-blaze-client                    2.12_0.22-1.0      java-archive                  
http4s-blaze-client                    2.12_0.23-1.0      java-archive                  
http4s-blaze-client                    2.13_0.21-1.0      java-archive                  
http4s-blaze-client                    2.13_0.22-1.0      java-archive                  
http4s-blaze-client                    2.13_0.23-1.0      java-archive                  
http4s-blaze-server                    2.12_0.21-1.0      java-archive                  
http4s-blaze-server                    2.12_0.22-1.0      java-archive                  
http4s-blaze-server                    2.12_0.23-1.0      java-archive                  
http4s-blaze-server                    2.13_0.21-1.0      java-archive                  
http4s-blaze-server                    2.13_0.22-1.0      java-archive                  
http4s-blaze-server                    2.13_0.23-1.0      java-archive                  
http4s-ember-client                    2.12_0.23-1.0      java-archive                  
http4s-ember-client                    2.13_0.23-1.0      java-archive                  
http4s-ember-server                    2.12_0.23-1.0      java-archive                  
http4s-ember-server                    2.13_0.23-1.0      java-archive                  
httpclient                             3-1.0              java-archive                  
httpclient                             3.1-1.0            java-archive                  
httpclient                             4.0-1.0            java-archive  (+1 duplicate)  
httpclient                             5.0-1.0            java-archive  (+1 duplicate)  
httpclient-jdk11                       1.0                java-archive  (+1 duplicate)  
httpurlconnection                      1.0                java-archive                  
hystrix                                1.3.15-1.0         java-archive                  
hystrix                                1.4-1.0            java-archive                  
java-io                                1.0                java-archive                  
java-io-inputstream-jdk8               1.0                java-archive                  
java-io-inputstream-jdk9               1.0                java-archive                  
java-io-stream                         1.0                java-archive                  
java-lang                              1.0                java-archive                  
javax-jndi                             1.0                java-archive                  
javax-ldap                             1.0                java-archive                  
javax-xpath                            1.0                java-archive                  
jax-rs                                 1.0-1.0            java-archive  (+1 duplicate)  
jax-rs                                 3.0-1.0            java-archive  (+1 duplicate)  
jaxen-xpath                            1.0                java-archive                  
jaxen-xpath                            1.1-1.0            java-archive                  
jboss                                  7-1.0              java-archive                  
jboss-jmx                              7-1.0              java-archive                  
jboss-logging                          1.0                java-archive                  
jcache                                 1.0.0-1.0          java-archive                  
jcache-datastore                       1.0.0-1.0          java-archive                  
jdbc-db2                               1.0                java-archive  (+1 duplicate)  
jdbc-driver                            1.0                java-archive                  
jdbc-embedded-derby                    10.10.1.1-1.0      java-archive                  
jdbc-embedded-derby                    10.11.1.1-1.0      java-archive  (+1 duplicate)  
jdbc-embedded-derby                    10.15.1.3-1.0      java-archive  (+1 duplicate)  
jdbc-embedded-derby                    10.2.1.6-1.0       java-archive                  
jdbc-generic                           1.0                java-archive  (+1 duplicate)  
jdbc-h2                                1.0                java-archive  (+1 duplicate)  
jdbc-hsqldb                            1.7.2.2-1.0        java-archive  (+1 duplicate)  
jdbc-hsqldb                            2.2.9-1.0          java-archive  (+1 duplicate)  
jdbc-inet-merlia                       1.0                java-archive  (+1 duplicate)  
jdbc-inet-oranxo                       1.0                java-archive  (+1 duplicate)  
jdbc-jtds                              1.0                java-archive                  
jdbc-jtds-generic                      1.0                java-archive  (+1 duplicate)  
jdbc-mariadb                           1.1.7-1.0          java-archive  (+1 duplicate)  
jdbc-mariadb                           1.3.0-1.0          java-archive  (+1 duplicate)  
jdbc-mysql                             3.0.8-1.0          java-archive  (+1 duplicate)  
jdbc-mysql                             6.0.2-1.0          java-archive  (+1 duplicate)  
jdbc-mysql                             6.0.4-1.0          java-archive                  
jdbc-mysql                             8.0.11-1.0         java-archive  (+1 duplicate)  
jdbc-mysql-connection                  5.1.4-1.0          java-archive  (+1 duplicate)  
jdbc-mysql-connection                  6.0.2-1.0          java-archive  (+1 duplicate)  
jdbc-mysql-connection                  8.0.11-1.0         java-archive  (+1 duplicate)  
jdbc-mysql-multihost-connections       5.1.3-1.0          java-archive  (+1 duplicate)  
jdbc-mysql-multihost-connections       6.0.2-1.0          java-archive  (+1 duplicate)  
jdbc-mysql-multihost-connections       8.0.11-1.0         java-archive                  
jdbc-ojdbc                             1.0                java-archive                  
jdbc-ojdbc-tns                         1.0                java-archive                  
jdbc-ojdbc-tns                         12.1.0.2-1.0       java-archive                  
jdbc-ojdbc-tns                         21.1.0.0-1.0       java-archive                  
jdbc-ojdbc-tns                         21.6.0.0-1.0       java-archive                  
jdbc-ojdbc7                            12.1.0.2-1.0       java-archive  (+1 duplicate)  
jdbc-ojdbc8                            21.1.0.0-1.0       java-archive  (+1 duplicate)  
jdbc-postgresql                        8.0-312.jdbc3-1.0  java-archive  (+1 duplicate)  
jdbc-postgresql                        9.4.1207-1.0       java-archive  (+1 duplicate)  
jdbc-postgresql                        9.4.1208-1.0       java-archive  (+1 duplicate)  
jdbc-resultset                         1.0                java-archive                  
jdbc-socket                            1.0                java-archive                  
jdbc-socket-jdk21                      1.0                java-archive                  
jdbc-sqlserver                         1.0                java-archive  (+1 duplicate)  
jdbc-sybase                            6-1.0              java-archive  (+1 duplicate)  
jedis                                  1.4.0-1.0          java-archive                  
jedis                                  2.7.1-1.0          java-archive                  
jedis                                  2.7.2-1.0          java-archive                  
jedis                                  3.0.0-1.0          java-archive                  
jedis                                  4.0.0-1.0          java-archive                  
jersey                                 1-1.0              java-archive                  
jersey                                 2-1.0              java-archive                  
jersey                                 3-1.0              java-archive                  
jersey-client                          1-1.0              java-archive                  
jersey-client                          2-1.0              java-archive                  
jersey-client                          2.26-1.0           java-archive                  
jersey-client                          3-1.0              java-archive                  
jetty                                  10-1.0             java-archive                  
jetty                                  11-1.0             java-archive  (+1 duplicate)  
jetty                                  12-1.0             java-archive                  
jetty                                  9-1.0              java-archive                  
jetty                                  9.3-1.0            java-archive                  
jetty-ee10-servlet                     12-1.0             java-archive                  
jetty-ee10-servlet                     12.0.1-1.0         java-archive                  
jetty-ee8-servlet                      12-1.0             java-archive                  
jetty-ee9-servlet                      12-1.0             java-archive                  
jetty-jmx                              7-1.0              java-archive                  
jetty-servlet                          9-1.0              java-archive                  
jms                                    1.1-1.0            java-archive                  
jms                                    3-1.0              java-archive                  
jsp                                    2.4-1.0            java-archive  (+1 duplicate)  
jsp                                    3-1.0              java-archive  (+1 duplicate)  
kafka-clients-config                   1.1.0-1.0          java-archive                  
kafka-clients-heartbeat                0.10.1.0-1.0       java-archive                  
kafka-clients-heartbeat                2.1.0-1.0          java-archive                  
kafka-clients-metrics                  0.10.0.0-1.0       java-archive                  
kafka-clients-metrics                  2.0.0-1.0          java-archive                  
kafka-clients-metrics                  3.0.0-1.0          java-archive                  
kafka-clients-node-metrics             1.0.0-1.0          java-archive                  
kafka-clients-spans                    0.11.0.0-1.0       java-archive                  
kafka-connect-metrics                  1.0.0-1.0          java-archive                  
kafka-connect-spans                    2.0.0-1.0          java-archive                  
kafka-connect-spans                    3.3.0-1.0          java-archive                  
kafka-streams-metrics                  1.0.0-1.0          java-archive                  
kafka-streams-spans                    2.0.0-1.0          java-archive                  
kafka-streams-spans                    2.1.0-1.0          java-archive                  
kafka-streams-spans                    2.6.0-1.0          java-archive                  
kafka-streams-spans                    3.2.0-1.0          java-archive                  
ldaptive                               1.0-1.0            java-archive                  
ldaptive                               2.0-1.0            java-archive                  
lettuce                                4.3-1.0            java-archive                  
lettuce                                5.0-1.0            java-archive                  
lettuce                                6.0-1.0            java-archive                  
logback-classic                        1.2-1.0            java-archive                  
logging-jdk8                           1.0                java-archive                  
low-priority-instrumentation           1.0                java-archive                  
mongodb                                3.0-1.0            java-archive                  
mongodb                                3.1-1.0            java-archive                  
mongodb                                3.6-1.0            java-archive                  
mongodb                                3.7-1.0            java-archive  (+1 duplicate)  
mongodb                                3.8-1.0            java-archive                  
mongodb-async                          3.10-1.0           java-archive                  
mongodb-async                          3.11-1.0           java-archive                  
mongodb-async                          3.4-1.0            java-archive                  
mongodb-async                          3.6-1.0            java-archive                  
mongodb-async                          3.7-1.0            java-archive                  
mongodb-async                          3.8-1.0            java-archive                  
mongodb-async                          4.0-1.0            java-archive                  
mongodb-reactive-streams               4.2-1.0            java-archive                  
mongodb-reactive-streams               4.8-1.0            java-archive                  
monix                                  2.13_3.4.0-1.0     java-archive                  
mule                                   3.6-1.0            java-archive                  
mule                                   3.7-1.0            java-archive                  
mule-base                              1.0                java-archive                  
mule-ei1                               1.0                java-archive                  
mule-ei2                               1.0                java-archive                  
nashorn-jsinjection                    1.0                java-archive                  
netty                                  3.4-1.0            java-archive                  
netty                                  3.8-1.0            java-archive                  
netty                                  4.0.0-1.0          java-archive                  
netty                                  4.0.8-1.0          java-archive                  
netty-reactor                          0.7.0-1.0          java-archive                  
netty-reactor                          0.8.0-1.0          java-archive                  
netty-reactor                          0.9.0-1.0          java-archive                  
newrelic                               8.9.1              java-archive                  
newrelic-api                           8.9.1              java-archive  (+1 duplicate)  
newrelic-api-javadoc                                      java-archive                  
newrelic-api-sources                                      java-archive                  
newrelic-cats-effect3-api              1.0                java-archive                  
newrelic-scala-cats-api                1.0                java-archive                  
newrelic-scala-monix-api               1.0                java-archive                  
newrelic-scala-zio-api                 1.0                java-archive                  
newrelic-security-agent                1.1.0              java-archive                  
newrelic-security-api                  1.1.0              java-archive                  
newrelic-weaver-api                    8.9.1              java-archive                  
newrelic-weaver-scala-api              8.9.1              java-archive                  
ning-async-http-client                 1.0-1.0            java-archive                  
ning-async-http-client                 1.1-1.0            java-archive                  
ning-async-http-client                 1.6.1-1.0          java-archive                  
okhttp                                 3.0.0-1.0          java-archive                  
okhttp                                 3.14.0-1.0         java-archive                  
okhttp                                 3.5.0-1.0          java-archive                  
okhttp                                 3.6.0-1.0          java-archive                  
okhttp                                 4.0.0-1.0          java-archive  (+1 duplicate)  
okhttp                                 4.4.0-1.0          java-archive                  
open-liberty-jakarta                   21.0.0.12-1.0      java-archive                  
openejb                                3.0-1.0            java-archive                  
openjpa                                1.0.0-1.0          java-archive                  
oracle-r2dbc                           1.0                java-archive                  
play                                   2.3-1.0            java-archive                  
play                                   2.4-1.0            java-archive                  
play                                   2.5-1.0            java-archive                  
play                                   2.6-1.0            java-archive                  
play                                   2.6.13-1.0         java-archive                  
play                                   2.7-1.0            java-archive                  
play                                   2.8.16-1.0         java-archive                  
play-shaded-async-http-client          1.0.0-1.0          java-archive                  
play-ws                                2.13_2.6.0-1.0     java-archive                  
play-ws                                2.6.0-1.0          java-archive                  
process                                1.0                java-archive                  
quartz                                 1.8.3-1.0          java-archive                  
quartz                                 2.0.0-1.0          java-archive                  
r2dbc-generic                          1.0                java-archive                  
r2dbc-h2                               1.0                java-archive  (+1 duplicate)  
r2dbc-mariadb                          1.0                java-archive  (+1 duplicate)  
r2dbc-mssql                            1.0                java-archive  (+1 duplicate)  
r2dbc-mysql                            1.0                java-archive  (+1 duplicate)  
r2dbc-oracle                           1.0                java-archive                  
r2dbc-postgresql                       0.9.0-1.0          java-archive                  
r2dbc-postgresql                       0.9.2-1.0          java-archive                  
r2dbc-postgresql                       1.0                java-archive                  
rabbit-amqp                            2.7-1.0            java-archive                  
rabbit-amqp                            3.5.0-1.0          java-archive                  
rabbit-amqp                            5.0.0-1.0          java-archive                  
reactor                                3.3.0-1.0          java-archive                  
resin                                  3-1.0              java-archive                  
resin                                  4-1.0              java-archive                  
resin-jmx                              1.0                java-archive                  
resteasy                               2.2-1.0            java-archive                  
resteasy                               3.0-1.0            java-archive                  
rhino-jsinjection                      1.0                java-archive                  
saxpath                                1.0                java-archive                  
scala                                  2.12.0-1.0         java-archive                  
scala                                  2.13.0-1.0         java-archive                  
scala                                  2.9.3-1.0          java-archive                  
servlet                                2.4-1.0            java-archive  (+1 duplicate)  
servlet                                5.0-1.0            java-archive  (+1 duplicate)  
servlet                                6.0-1.0            java-archive  (+1 duplicate)  
servlet-user                           1.0                java-archive                  
servlet-user                           5.0-1.0            java-archive                  
slick                                  2.11_3.2.0-1.0     java-archive                  
slick                                  2.12_3.2.0-1.0     java-archive                  
slick                                  3.0.0-1.0          java-archive                  
solr                                   4.0.0-1.0          java-archive                  
solr                                   5.0.0-1.0          java-archive                  
solr                                   5.1.0-1.0          java-archive                  
solr                                   5.4.0-1.0          java-archive                  
solr                                   6.4.0-1.0          java-archive                  
solr                                   7.0.0-1.0          java-archive                  
solr-jmx                               7.0.0-1.0          java-archive                  
solr-jmx                               7.4.0-1.0          java-archive                  
spray-can                              1.3.1-1.0          java-archive                  
spray-can-http-client                  1.3.1-1.0          java-archive                  
spray-client                           1.3.1-1.0          java-archive                  
spray-http                             1.3.1-1.0          java-archive                  
spring                                 3.0.0-1.0          java-archive                  
spring                                 4.0.0-1.0          java-archive                  
spring                                 4.2.0-1.0          java-archive                  
spring                                 4.3.0-1.0          java-archive                  
spring                                 6.0.0-1.0          java-archive                  
spring-aop                             2-1.0              java-archive                  
spring-cache                           3.1.0-1.0          java-archive                  
spring-jms                             2-1.0              java-archive                  
spring-jms                             3-1.0              java-archive                  
spring-web                             1.0                java-archive                  
spring-webclient                       5.0-1.0            java-archive                  
spring-webclient                       6.0-1.0            java-archive                  
spring-webflux                         5.0.0-1.0          java-archive                  
spring-webflux                         5.1.0-1.0          java-archive                  
spring-webflux                         5.3.0-1.0          java-archive                  
spring-webflux                         6.0.0-1.0          java-archive                  
spring-webflux-controller-mappings     5.0.0-1.0          java-archive                  
spring-ws                              2.0-1.0            java-archive                  
spymemcached                           2.12.0-1.0         java-archive                  
sttp                                   2.12_2.2.3-1.0     java-archive                  
sttp                                   2.13_2.2.3-1.0     java-archive                  
sttp                                   2.13_3.0.0-1.0     java-archive                  
sttp-akka                              2.12_2.1.5-1.0     java-archive                  
sttp-akka                              2.13_2.1.5-1.0     java-archive                  
sttp-akka                              2.13_3.0.0-1.0     java-archive                  
sttp-http4s                            2.12_2.1.5-1.0     java-archive                  
sttp-http4s                            2.13_2.1.5-1.0     java-archive                  
sttp-http4s                            2.13_3.3.0-1.0     java-archive                  
sun-net-httpserver                     1.0                java-archive  (+1 duplicate)  
thrift                                 0.8-1.0            java-archive                  
tomcat                                 10-1.0             java-archive                  
tomcat                                 7-1.0              java-archive  (+1 duplicate)  
tomcat                                 7.0.70-1.0         java-archive                  
tomcat                                 8-1.0              java-archive                  
tomcat                                 8.5.0-1.0          java-archive                  
tomcat                                 8.5.2-1.0          java-archive                  
tomcat-jmx                             1.0                java-archive                  
tomcat-request-listener                1.0                java-archive                  
tomcat-request-listener                10-1.0             java-archive                  
unboundid-ldapsdk                      1.0                java-archive                  
urlconnection                          1.0                java-archive                  
vertx-core                             3.3.0-1.0          java-archive                  
vertx-core                             3.3.3-1.0          java-archive                  
vertx-core                             3.4.1-1.0          java-archive                  
vertx-core                             3.6.0-1.0          java-archive                  
vertx-core                             3.8.0-1.0          java-archive                  
vertx-core                             3.9.0-1.0          java-archive                  
vertx-core                             4.0.0-1.0          java-archive                  
vertx-core                             4.1.0-1.0          java-archive                  
vertx-core                             4.3.2-1.0          java-archive                  
vertx-core                             4.5.0-1.0          java-archive                  
vertx-core                             4.5.1-1.0          java-archive                  
vertx-web                              3.2.0-1.0          java-archive                  
vertx-web                              3.3.0-1.0          java-archive                  
vertx-web                              3.5.0-1.0          java-archive                  
vertx-web                              3.5.1-1.0          java-archive                  
vertx-web                              3.5.2-1.0          java-archive                  
vertx-web                              3.6.0-1.0          java-archive                  
vertx-web                              3.8.0-1.0          java-archive                  
vertx-web                              3.8.3-1.0          java-archive                  
weblogic                               12-1.0             java-archive                  
weblogic                               12.2-1.0           java-archive                  
weblogic-jmx                           12-1.0             java-archive                  
weblogic-jmx                           12.2.1-1.0         java-archive                  
websphere                              8-1.0              java-archive                  
websphere-jmx                          7-1.0              java-archive                  
websphere-liberty-profile              8.5.5.5-1.0        java-archive                  
websphere-liberty-profile-dispatcher   8.5.5.5-1.0        java-archive                  
websphere-liberty-profile-environment  8.5.5.5-1.0        java-archive                  
wildfly                                27-1.0             java-archive                  
wildfly                                8-1.0              java-archive                  
wildfly                                8-CAT-1.0          java-archive                  
wildfly                                8-PORT-1.0         java-archive                  
wildfly-jmx                            14-1.0             java-archive                  
xalan-xpath                            1.0                java-archive                  
xml                                    1.0                java-archive  (+1 duplicate)  
zio                                    1.0                java-archive  
```

Reply:
Yes, there are currently no known public vulnerabilities filed against the Newrelic java agent components

Reply:
and there are currently no separate maven components published for the instrumentation jars, so when there are vulns in future they would likely be placed against one or multiple of the published maven jars which we are still capturing (though probably the groupie and artifact I'd may be incorrect, but that is a separate issue)